great-idea/css/index.css

```diff
@@ -62,3 +62,151 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+.container {
+	padding:5%;
+	padding-top:2.5%;
+}
+
+header{
+	display:flex;
+	/* outline: red 2px solid; */
+	justify-content: space-between;
+	align-items:center;
+}
+
+header nav {
+	/* display: inline-block;
+	position:relative;
+	width:80%; */
+	display:flex;
+	text-decoration: none;
+	/* outline: blue 2px solid; */
+	justify-content: space-between;
+	align-items:flex-end;
+	width:70%;
+}
+
+header nav a {
+	/* display: inline-block;
+	vertical-align: top;
+	float: left;
+	color:grey;
+	padding:25px;
+	padding-right:70px; */
+	/* outline: green 2px solid; */
+	color: grey;
+	text-decoration: none;
+}
+
+header img{
+	/* display: inline-block;
+	vertical-align: top;
+	padding-top: 10px;
+	float: right;
+	/* width:20%; */
+}
+
+.cta{
+    display:flex;
+    /* position:relative; */
+    border-bottom: gray 2px solid;
+    /* outline: red 2px solid; */
+    width:100%;
+		margin-bottom:20px;
+		justify-content: space-around;
+		padding:5% 0 5% 0;
+}
+.cta .cta-text {
+    /* vertical-align: middle;
+    width: 50%; */
+		display:flex;
+		flex-direction: column;
+		text-align: center;
+		align-items:center;
+		justify-content:center;
+}
+.cta h1 {
+    /*display:inline-block;*/
+    font-size: 500%;
+		/* padding-top: 20%;
+    padding-right: 20px; */
+    /* outline: yellow 2px solid; */
+}
+.cta button{
+    /* outline: yellow 2px solid; */
+		border: black 2px solid;
+		padding: 10px 45px;
+		font: inherit;
+}
+.cta img{
+    /* outline: yellow 2px solid; */
+    /* float: right;
+    width:50%;
+		padding:5%; */
+		align-self: flex-end;
+}
+
+.main-content{
+    display:flex;
+    /* position:relative; */
+    border-bottom: gray 2px solid;
+    /* outline: red 2px solid; */
+    width:100%;
+		margin-bottom:20px;
+		flex-flow: row wrap;
+}
+
+ .main-content .top-content {
+		display:flex;
+		justify-content: space-between;
+}
+
+.main-content .top-content .text-content{
+    width:48.5%;
+		/* float:left; */
+		padding-top: 40px;
+		padding-bottom:40px;
+		/* padding-right: 5px; */
+    outline: red 2px solid;
+}
+
+.main-content .middle-img{
+		width:100%;
+	/* padding-left:3%;
+	padding-right:3%; */
+}
+
+.main-content .bottom-content {
+	display:flex;
+	justify-content: space-between;
+}
+
+.main-content .bottom-content .text-content{
+    width:30%;
+		padding-top: 40px;
+		padding-bottom:40px;
+    /* outline: red 2px solid; */
+}
+
+.contact p{
+	display:flex;
+	/* outline: red 2px solid; */
+}
+/*
+.contact p:after {
+	content:'\a';
+	white-space: pre;
+} */
+
+footer{
+	/* outline:red 2px solid; */
+	display:flex;
+	align-items: flex-end;
+	justify-content: center;
+}
+
+footer:before{
```

Comment:
Amazing work exploring psuedo selectors, I was apprehensive to introduce this to you for the inline version. I'm impressed.